### ISSUE TITLE
Two-factor authentication for Remotes in GitHub

### ISSUE THREAD

**Issue by Copepoda** [swcarpentry/git-novice#564]
In the Remotes in GitHub chapter please add a note for those of us who use two factor authentication. Where we need to use a personal access token instead of the regular password. 

https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/

Thank you. :)

**Comment by rgaiacs** [comment]
Hi @Copepoda Thanks for the issue. I have mixed feelings about include this because if you are using GitHub for the first time, why do you have two-factor authentication? And if you are one of the instructors or helpers that are using two-factor authentication, why do you don't know this already?

**Comment by Copepoda** [comment]
Hi @rgaiacs fair point. I guess I come from a perspective of, if 2-factor is available that you use it. I really thought the lessons plans were great!

**Comment by rgaiacs** [comment]
> I guess I come from a perspective of, if 2-factor is available that you use it.

I agree with you. But given that in the lesson we don't teach learners how to use SSH instead of HTTPS to pull/pull data to Git servers given our time constrains, I would say that we don't have time to teach two-factor authentication. Said that, are you OK to close this issue? Thanks for raise it.
